jstz_cli/src/main.rs

```diff
@@ -26,10 +27,10 @@ enum Command {
         #[arg(short, long)]
         self_address: String,
         /// Function code.
-        #[arg(short, long)]
-        function_code: String,
+        #[arg(short, long, default_value = None)]
+        function_code: Option<String>,
```

Comment:
It would be nice if this wasn't a flagged argument anymore, but the only unnamed positional argument